src/util/commandline/command.ts

```diff
@@ -113,6 +115,13 @@ export class Command {
       setQuiet();
     }
 
+    // Check if a new cli version is available
+    const result: Buffer = execSync("npm show appcenter-cli version");
```

Comment:
We should catch and output any errors here - if something goes wrong checking the version - we don't want to stop the cli from working.